controller/service.go

```diff
@@ -185,8 +208,24 @@ func (c *controller) allocateIPs(key string, svc *v1.Service) ([]net.IP, error)
 		return nil, err
 	}
 
-	// If the user asked for a specific IP, try that.
-	if svc.Spec.LoadBalancerIP != "" {
+	var lbIPs []net.IP
+	// annotationLoadBalancerIPs takes higher precedence over svc.Spec.LoadBalancerIP
+	// TODO: remove svc.Spec.LoadBalancerIP check when this field is removed
+	// from service spec.
+	desiredLbIPs, err := c.getDesiredLbIPs(svc)
```

Comment:
Same as above, incorporate `spec.LoadBalancerIP` in getDesiredLBIPs

Reply:
done